scripts/teamDataScript.js

```diff
@@ -0,0 +1,435 @@
+    
+    //either one or both of eventChecked or isSeasonSaved() is false, show modal showing one or more of 
+    if (!isSeasonSaved())
+    {
+        modalContentHTML = '<p>No season is currently checked. Go to the <a href="season-config.html">Season Configuration</a> page '+ 
+        'to enter new seasons and season-specific elements/pit scouting questions.' + 
+        '<br><br>Make sure you check the new season you enter.</p>';
+    }
+
+    if (!eventChecked)
+    {
+        //just for formatting: looks weird if there is a <br><br> before the ev checked message if the season checked message isn't there before it
+        var preMessageSpace = isSeasonSaved() ? '' : '<br><br>';
+    
+        modalContentHTML += preMessageSpace + '<p>No event is currently checked. Go to the <a href="event-data.html">Event Data</a> page '+ 
+        'to enter new events and teams attending those events for a selected season.' + 
+        '<br><br>Make sure you check the new event you enter.</p>';
+    }
+    
+    $('#tDataModalContent').append(modalContentHTML);
+    $('#teamDataAlertModal').show();
+}
+
+function closeTDataModal() //onclick for x button in modal
+{
+    $('#teamDataAlertModal').hide();
+}
+
+function displayTeamRadioList()
+{
+    var season = getCurrSeason();
+    var event = getCurrEvent();
+    if (season.trim().length == 0 || event.trim().length == 0) return;
+
+    var teamsJSON = getTeams(season,event);
+    var keys = Object.keys(teamsJSON);
+    for (var i = 0; i < keys.length; i++)
+    {
+        var teamNumber = keys[i];
+        var teamName = teamsJSON[teamNumber];
+
+        var newItem = `<label><input type='radio' name='teamListItem' value='${teamNumber}'>${teamNumber}: ${teamName}</label><br>`;
+        $("#teamRadioList").append(newItem); //add to radiolist
+    }
+}
+
+//onchange for the team radio list
+function changeCurrTeam()
+{
+    var currTeamNumber = $("input[name=teamListItem]:checked","#teamRadioList").val(); //in prev method, val of each radio item is the tNum
+    var currSeason = getCurrSeason();
+    var currEvent = getCurrEvent();
+    var cname = 'currCheckedTeam';
+    var currTeamName = getTeams(currSeason,currEvent)[currTeamNumber];
+
+    if (currSeason.trim().length == 0 || currEvent.trim().length==0 || currTeamNumber == 0)
+    {
+        $("#teamStatsTitle").text('');
+        return;
+    }
+
+    var cvalue = `/${currSeason}/${currEvent}/${currTeamNumber}`;
+    setCookie(cname,cvalue,750);     //curr team cookie --> currCheckedTeam = /{season}/{event}/{currTeamNumber}
+
+    $("#teamStatsTitle").text(`Data for team ${currTeamNumber} : ${currTeamName}`);
+    $('#pitScoutingTitle').text(`Pit Scouting Questions and Answers for team ${currTeamNumber}: ${currTeamName}`);
+    $('.js_clear_on_load').val("").html("");
+
+    loadPit();
+    loadMatch();
+}
+
+//basically copy pasted from w3schools, onclick for a change in tabs
+function displayTabContent(evt, tabName)
+{
+    var i, x, tablinks;
```

Comment:
I don't think you're using any of these variables anymore. (iff I'm right, remove them).